### ISSUE TITLE
Speed is too fast, no way to configure it to be lower than +7 Speed

### ISSUE THREAD

**Issue by Nomelas** [JessebotX/StardewMods#5]
**Mod you are referring to**:
Sprint Sprint Sprint

**Singleplayer or Multiplayer**:
Both

**Reproducible**: 

Fresh install of the mod and load for the first time.

**Description**:

Default speed in config.conf is 5, which, in-game is +12 speed! this is wayy to fast!!! If I lower the setting in the config to 0, it's still +7, which is faster than the advertised +5 speed! Is this a bug? I can set it to -1 and -2 etc, but it really glitches out and I go super fast lol.

**Provide SMAPI Log**:

**Comment by Nomelas** [comment]
Tried in both 1.0.3 and 1.0.4 where i left the speed default, and it's always +12 speed.... Sprint is the only mod installed, so there's no way its a conflict.

SMAPI Version 2.11.2

**Comment by JessebotX** [comment]
I will look into an option to disable that

**Comment by JessebotX** [comment]
just to let you know im now looking to add an option to disable the +7 speed when u have the Sprint key on leftshift. The reason being is that left shift is also the run key in vanilla game and since auto run is enabled by default, it slows the player down

**Comment by JessebotX** [comment]
I have fixed the issue, however I am planning to add more before a stable release build is released on Nexus. I have uploaded a prerelease build that has been tested on github instead so you don't have to wait any longer. Download ```SprintSprintSprint-1.0-beta.20190816``` and run the game once to reset the config. In the config, set LeftShiftKeyExtraSpeed to something lower if you find the added speed too much.

[Download on github](https://github.com/JessebotX/StardewMods/releases/tag/Sprint)
[Release Notes - will be updated shortly](https://github.com/JessebotX/StardewMods/blob/master/Sprint/changelog.md)
